internal/authtoken/repository.go

```diff
@@ -33,13 +33,14 @@ type Repository struct {
 // NewRepository creates a new Repository. The returned repository is not safe for concurrent go
 // routines to access it.
 func NewRepository(r db.Reader, w db.Writer, kms *kms.Kms, opt ...Option) (*Repository, error) {
+	const op = "authtoken.NewRepository"
 	switch {
 	case r == nil:
-		return nil, fmt.Errorf("db.Reader: auth token: %w", errors.ErrInvalidParameter)
+		return nil, errors.New(errors.InvalidParameter, op, "missing db reader")
```

Comment:
This is fine, but for some reason, the standard for these specific errors in other places have been `nil db reader`, `nil db writer` and `nil kms`